modules/apps/forms-and-workflow/portal-workflow/portal-workflow-web/src/main/resources/META-INF/resources/css/_definition.scss

```diff
@@ -11,6 +11,7 @@
 }
 
 .info-bar-item {
+	margin-top: -3px;
```

Comment:
Hey @jeyvison, why did you put a negative margin in the .info-bar-item component? Let me know your thinking. Depending of the context, you can put a negative top property, like: `top: -3px`, but it's not a good practice to use negative measures of margins or positions.

Reply:
Hey @clovisdasilvaneto , I talked with Matusalem about it when we decided to go this way but i have no problem on changing it if the result is the same.